### ISSUE TITLE
Configure location of Info Panel?

### ISSUE THREAD

**Issue by Drecula** [airbnb/react-dates#840]
Hi,

Might we add a config so the Info Panel could be Top, Right, Bottom, or Left vs always at the bottom?

Thanks!

**Comment by majapw** [comment]
Yes! I meant to respond to this when this was first opened, but this seems like a great idea.

**Comment by varun1101** [comment]
I intend to start on this. Just wanted to know should this appear as a separate filter in Storybook or as an manual option similar to animation etc.

**Comment by majapw** [comment]
Hi @varun1101, 

I think ideally that the DateRangePicker and SingleDatePicker would take a `calendarInfoPosition` prop and that it would behave very similarly to the `inputIconPosition` prop. It would be good to add like `INFO_POSITION_TOP`, `INFO_POSITION_BOTTOM`, `INFO_POSITION_LEFT`, and `INFO_POSITION_RIGHT` to the constants file and use those as enums. 

I would just add examples to https://github.com/airbnb/react-dates/blob/master/stories/DateRangePicker_calendar.js and https://github.com/airbnb/react-dates/blob/master/stories/SingleDatePicker_calendar.js in much the same way as the 'with info panel' story.

**Comment by varun1101** [comment]
Thanks @majapw for the help.

**Comment by mariepw** [comment]
Hi, 
Any updates on this? 
@varun1101 did you work on something? Or you want me to do the PR? 
Thanks.

**Comment by varun1101** [comment]
Hi,
I did not get any time in particular to work on this. You can go ahead with it @mariepw . Thanks

**Comment by Airboy** [comment]
Hi @mariepw,

Would be nice to update the documentation about the use of this feature ;-)
`calendarInfoPosition` is missing within the README file. Let me know if I can help with that.
Thanks.

**Comment by mariepw** [comment]
@Airboy you're right, thanks. I'll try to make a PR to add it during the week-end :) Or you can do it if you want to ..
